images/runtime/node/installDependencies.sh

```diff
@@ -36,8 +36,8 @@ if [[ $nodeVersionMajor -ge 10  ]]; then
     echo "Version of npm after upgrade: $currentNpmVersion"
 fi
 
-# Do NOT install PM2 from Node 14 onwards
-if [[ $nodeVersionMajor -lt 14 ]]; then
+# Do NOT install PM2 for Node versions 8 older than 8
```

Comment:
nit: I was a little confused by this comment due to the typo -- I would reword it to be something like:

_Do NOT install PM2 for Node versions below 8 (older than 8)_